app/src/ui/lib/text-box.tsx

```diff
@@ -132,10 +149,8 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
     })
   }
 
-  private onRef = (instance: HTMLInputElement | null) => {
-    if (this.props.onInputRef) {
-      this.props.onInputRef(instance)
-    }
+  private inputRef = (element: HTMLInputElement) => {
```

Comment:
I think this should be `element: HTMLInputElement | null` right?

Reply:
Yes, according to their docs, `null` is passed when the component unmounts. Good catch!